### PR TITLE
bump TK/HarfBuzz/at_spi2 for alignment

### PR DESCRIPTION
**Pull request by vchuravy**
See #760

### CODE REVIEW

File: H/HarfBuzz/build_tarballs.jl [hunk 1]
```diff
@@ -39,13 +39,14 @@ products = [
...
-    "Fontconfig_jll",
-    "Cairo_jll",
+    Dependency("Glib_jll"),
+    Dependency("FreeType2_jll"),
+    Dependency("Graphite2_jll"),
+    Dependency("Libffi_jll"),
+    Dependency("Gettext_jll"),
+    Dependency("Fontconfig_jll"),
+    Dependency("Cairo_jll"),
+    BuildDependency("Xorg_kbproto_jll")
```
Comment (giordano) [inline H/HarfBuzz/build_tarballs.jl:49]: Better to use
```suggestion
    BuildDependency("Xorg_xorgproto_jll")
```

File: H/HarfBuzz/build_tarballs.jl [hunk 2]
```diff
@@ -39,13 +39,14 @@ products = [
...
-    "Gettext_jll",
-    "Glib_jll",
-    "FreeType2_jll",
-    "Fontconfig_jll",
-    "Cairo_jll",
+    Dependency("Glib_jll"),
+    Dependency("FreeType2_jll"),
+    Dependency("Graphite2_jll"),
+    Dependency("Libffi_jll"),
+    Dependency("Gettext_jll"),
```
Comment (giordano) [inline H/HarfBuzz/build_tarballs.jl:46]: Just speaking out loud.  I've never verified my hypothesis, but I think that also `Gettext_jll` is usually a build-only dependency.  We can leave as it is now, but I'd like to revise this at a later point.

Comment (giordano) [reply H/HarfBuzz/build_tarballs.jl:46]: For example, none of the products of Harfbuzz is linked against libgettextlib.  I don't know if other files from this package are used though

Comment (vchuravy) [reply H/HarfBuzz/build_tarballs.jl:46]: Yeah I was trying to determine that and I was looking at the Archlinux list and the Gentoo lists and didn't come to a conclusive answer.

**Pull request merged**